index.html

```diff
@@ -272,13 +347,13 @@ <h2>Social media</h2>
           <p>This site uses cookies to keep track your checklist progress. The data are stored in your browser locally only.</p>
           <p>If you have questions or suggestions about this page, please
                <a href="https://github.com/LSSTDESC/pre-meeting-checklist/issues" title="link to GitHub">open an issue</a> or contact
-               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:agkim@lbl.gov?Subject=DESC%20Meeting%20Checklist" target="_top">Alex Kim</a>.</p>
+               <a href="https://yymao.github.io">Yao-Yuan Mao</a> or <a href="mailto:plazasmalagon@gmail.com?Subject=DESC%20Meeting%20Checklist" target="_top">Andrés Plazas</a>.</p>
```

Comment:
Once published, this website is world-viewable. Please double check with @plazas to make sure it's ok to include email address in a mailto link (which may result in bot collecting this email address).

Reply:
okay andres is okay with either a slack or an email mention; i've just gone with the former.